ui/src/app/pages/admin/user/admin-user-table.tsx

```diff
@@ -280,7 +280,8 @@ export const AdminUserTable = withUserProfile()(
                 frozenWidth='200px'
                 globalFilter={filter}
                 paginator={true}
-                rows={50}
+                rows={5}
+                rowsPerPageOptions={[5, 10, 50, 100]}
```

Comment:
I like `rowHover` used in Datable. Do you?

Reply:
I had not heard of that attribute. I tried it out, and I like it, but noticed an issue. The frozen column (Name) and the rest of the columns are technically in different tables. This means that on hover it does not highlight everything. We could probably make it work, but it would involve some tinkering. I am going to hold off for now, but thanks for bringing it up.
<img width="1008" alt="image" src="https://user-images.githubusercontent.com/24514630/165803181-deb2bee7-f593-4754-a645-3bf60f25a4ae.png">